lib/OperatingSystems/FreeBSD.php

```diff
@@ -0,0 +1,239 @@
+<?php
+/**
+ * @author Frank Karlitschek <frank@nextcloud.com>
+ *
+ * @license AGPL-3.0
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\ServerInfo\OperatingSystems;
+
+use bantu\IniGetWrapper\IniGetWrapper;
+
+/**
+ * Class FreeBSD
+ *
+ * @package OCA\ServerInfo\OperatingSystems
+ */
+class FreeBSD {
+
+	/** @var IniGetWrapper */
+	protected $phpIni;
+	
+	/**
+	 * FreeBSD constructor.
+	 *
+	 * @param IniGetWrapper $phpIni
+	 * @throws \Exception
+	 */
+	public function __construct(IniGetWrapper $phpIni) {
+		$this->phpIni = $phpIni;
+	}
+	
+	/**
+	 * @return bool
+	 */
+	public function supported(): bool {
+		return true;
```

Comment:
```suggestion
		return false;
```

because FreeBSD is not supported.

Reply:
Done